structlog/stdlib.py

```diff
@@ -141,6 +231,35 @@ def __call__(self, *args):
         return logging.getLogger(name)
 
 
+class PositionalArgumentsFormatter(object):
+    """
+    Applies stdlib-like string formatting to the `event` key with the arguments
+    in the `positional_args` key. This is populated by
+    `structlog.stdlib.BoundLogger` or can be manually set.
+
+    `positional_args` can be any iterable, but a dictionary as the single
+    element of the tuple is used instead of the tuple, to mantain compatibility
+    with the undocumented feature of stdlib logging.
+
```

Comment:
No empty line here please (the pep rule comes from the fact that Guido’s editor needed it to reflow comments m().